### PR TITLE
Update ColumnHeader.svelte

### PR DESCRIPTION
**Pull request by luke-hagar-sp**
Removing specified ts extensions. Import paths should not end with .ts, as the paths are no longer valid after transpilation.

Should address #204, I am also searching for other instances

### CODE REVIEW

File: src/lib/components/controls/navigationBar/NavigationBar.svelte [hunk 1]
```diff
@@ -42,7 +42,7 @@
   }
 
   // trigger scrollToLastItem when path changes
-  $: scrollToLastItem(path)
+  $: scrollToLastItem()
```
Comment (josdejong) [inline src/lib/components/controls/navigationBar/NavigationBar.svelte:45]: The path actually needs to be there as argument. It is an ugly hack to "trigger scrollToLastItem when path changes".

I'm open to better solutions, but at least let's keep it as it is for the time being and address that in a separate PR.

**Review by josdejong — CHANGES_REQUESTED** [review]
Looks good, thanks Luke! 

I made one inline comment, after that we can merge your PR.

File: src/lib/components/controls/navigationBar/NavigationBar.svelte (hunk 1 above)
Comment (luke-hagar-sp) [reply src/lib/components/controls/navigationBar/NavigationBar.svelte]: ahhhhhhhh, I understand, it's for the reactive trigger.

In this example, I think its better to have the unused param on the function.
I amended my PR

Comment (josdejong) [reply src/lib/components/controls/navigationBar/NavigationBar.svelte]: 👍 Agree I'll adjust a couple of similar cases too, and add `// eslint-disable-next-line @typescript-eslint/no-unused-vars`  above it to make the linter happy.

Comment (luke-hagar-sp) [reply src/lib/components/controls/navigationBar/NavigationBar.svelte]: Roger That

**Review by josdejong — APPROVED** [review]

**Comment by josdejong** [comment]
The fix is published now in `v0.11.2`. It is literally Friday 17:00 at my place, always a good moment for a release 😂. 

Thanks again Luke.

**Comment by luke-hagar-sp** [comment]
:D

Now that I am able to use this package in my project, I am BLOWN AWAY with how feature-rich it is. You have REALLY outdone yourself here.

**Comment by josdejong** [comment]
Thanks, good to hear 😊 !

**Pull request merged**